web/html/src/scripts/docker-javascript-lint.sh

```diff
@@ -5,7 +5,7 @@ zypper --non-interactive in susemanager-nodejs-sdk-devel
 cd /manager/web/html/src
 
 # link to node_modules from susemanager-nodejs-sdk-devel
-ln -s /usr/lib/node_modules/ node_modules
+ln -s /usr/lib64/node_modules/ node_modules
```

Comment:
As requested by @lneves12.

Unfortunately this is part of spacewalk-web so @cbosdo, you will need to rebuild it in your branch to test. Can you have a look?